src/pages/FAQ/FAQ.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import Box from "@mui/material/Box";
-import AccordionList from "../../components/AccordionList";
+import AccordionList from "../../components/AccordionList/AccordionList";
```

Comment:
🔧 Add an `index.jsx` file, and import like this: `import AccordionList from "../../components/AccordionList";`

Reply:
Addressed: commit ba67774be0fa48ace3814e246d359199099aef84